packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -190,6 +195,9 @@ const IconType = React.memo(({ platform, type, show_eu_related_content }: TIconT
             case 'synthetic':
                 return <Icon icon='IcMt5StandardPlatform' size={128} />;
             case 'all':
+                if (product === PRODUCT.ZEROSPREAD) {
+                    return <Icon icon='IcMt5ZeroSpread' size={128} />;
```

Comment:
wow..... the only difference between these icons is their names.... could have been easier to just map the prop instead of returning repeated icons everywhere...

Reply:
We have initiative to re-write all of our components which have issues. These will be taken care of starting from the next sprint.